mis_builder/static/src/components/mis_report_widget.xml

```diff
@@ -59,24 +64,26 @@
                         </tr>
                     </thead>
                     <tbody>
-                        <tr t-foreach="widget.mis_report_data.body" t-as="row">
-                            <td t-att="{'style': row.style}">
-                                <t t-esc="row.label" />
-                                <t t-if="row.description">
+                        <tr t-foreach="state.mis_report_data.body" t-as="body_row" t-key="body_row_index">
```

Comment:
Any particular reason to rename row in body_row? If not let's keep the previous value to minimize the diff: this facilitates ports across versions.